### ISSUE TITLE
Possibility to soften the bruto force on shared links with password or without

### ISSUE THREAD

**Issue by MrLoverLoverMMMM** [nextcloud/bruteforcesettings#450]
The protection block at to many requests pretty fast.
Especially on public links it would be nice to be able to soften that or to exclude special public links from the protection.
Especially if they have a password.

So yes after a few password attempts it can block the ip. But it my case even if the password is typed in correct it block after too many requests / downloads.

**Comment by DanScharon** [comment]
yes, currently the bruteforce protection is too quick in regards to public shares. I know this is meant to block trying to guess possible public URLs, but you just need 20 user who click on an expired share and their whole subnet is getting blocked
